### ISSUE TITLE
Accessibility testing v3.0.11: Terms of use on Welcome screen

### ISSUE THREAD

**Issue by weglmar** [StatCan/StatcanPublicationUAT#563]
When selecting terms of use from the Welcome screen, the Go back and terms of use labels are not used correctly. It says altogether. and the user cannot go back from this screen. 

Go back should be changed to a Close button and the user should be able to close this screen.

**Comment by weglmar** [comment]
@ForumG-SC Good morning Forum! It would be Bouton de fermeture

**Comment by weglmar** [comment]
Confirmed fixed on my end
@Rajat-Sen could you please confirm? Thank you!

**Comment by Rajat-Sen** [comment]
This is fixed! 

However, the screen flow is not right. Not sure, if you would like to create another ticket for this.

At present (close button > switch to French > Terms of use> content).

It should be (close button > Terms of use > switch to French > content).

**Comment by Rajat-Sen** [comment]
Also, we can close this ticket because the screen-flow issues are in the whole app. Letting it go as the user won't be stuck on this screen anymore.

**Comment by weglmar** [comment]
Created ticket #571 for screen flow. 
Closing since the issue in this ticket is resolved.
